HW03/pavel_mishchenko/spec/pavel_mishchenko_hw_03_t_03_spec.rb

```diff
@@ -0,0 +1,109 @@
+# frozen_string_literal: true
+
+require_relative 'spec_helper'
+require_relative '../pavel_mishchenko_hw_03_t_03'
+
+RSpec.describe Homework3 do
+  describe '#task3' do
+    subject { described_class.new.task3(log) }
+
+    context 'when log that has right output' do
```

Comment:
maybe it's better to use contexts : 'when log is valid' and the opposite one? in all such situations